clang/include/clang/CConv/Constraints.h

```diff
@@ -579,7 +579,7 @@ class Constraints {
 
   void editConstraintHook(Constraint *C);
 
-  std::pair<Constraints::ConstraintSet, bool> solve();
+  std::pair<std::set<VarAtom *>, bool> solve();
```

Comment:
Why this change? The solution previously returned unsolvable constraints; now you are returning unsolvable varatoms?